app/Repositories/Cts/ExaminerRepository.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Repositories\Cts;
+
+use App\Models\Cts\ExaminerSettings;
+
+class ExaminerRepository
```

Comment:
Is this bound as a singleton somewhere?

Reply:
Nope - doesn't really need to be I don't think?

Reply:
I only ask because you're using `resolve` in the test suits, and ideally, you only want one copy of the repository object existing throughout the application.

Reply:
Following the trend from the other CTS repositories - I think it is fine for now but might look to refactor all of the repositories created to date soon?

Reply:
Sounds good to me